checkout-example-advanced/README.md

```diff
@@ -1,5 +1,7 @@
 # Adyen [Online Payments](https://docs.adyen.com/online-payments) Integration Demo - Advanced Flow
 
+Checkout sample application using `Adyen.Web Drop-in/Components v6.x.x`,  (see [folder /_archive/v5](./_archive/v5) to access the previous version using `Adyen.Web Drop-in/Components v5.x.x`).
```

Comment:
the folder should be `/checkout-example-advanced/_archive/v5` I believe

Reply:
It is the same in the Java Advanced Checkout README, it is a relative URL at the end of the day